src/components/jump-to-banner/jump-to-banner.js

```diff
@@ -0,0 +1,29 @@
+const jumpToBanner = () => {
+	if (document.querySelector(".js-jump-to")) {
+		const jumpToBanners = document.querySelectorAll(".js-jump-to");
+
+		jumpToBanners.forEach((jumpToBanner)=> {
```

Comment:
do we need to iterate through the _banners_? if you think about it, all the logic is nicely scoped to the individual link.  you could just iterate through those without this forEach.